### ISSUE TITLE
music player does not play

### ISSUE THREAD

**Issue by Gero5** [mpogue2/SquareDesk#908]
version 1.0.5, Autostart playback ON
Sometimes the music player does not play at all. music can be selected the title is displayed but it does not play the music automatically 
neither does it start by pressing the space bar 
neither by pressing the dot key 
neither by pressing play button.

restarting SquareDesk solves the problem, but it is annoying. 
It might have to do with waking up after the computer was in sleep mode.

**Comment by mpogue2** [comment]
This looks likes a dup of #808...

**Comment by Gero5** [comment]
yes except for the VU meter. :-) I already forgot that report, 100 issues have gone by since.
